tck/logs/src/main/java/org/eclipse/microprofile/telemetry/logs/tck/application/JulTest.java

```diff
@@ -49,13 +55,48 @@ public static WebArchive createDeployment() {
 
     private static final Logger julLogger = Logger.getLogger("jul-logger");
 
+    private static final String logFilePath = System.getProperty("log.file.path");
+
+    private static final String JUL_INFO_MESSAGE = "a very distinguishable info message";
+    private static final String JUL_WARN_MESSAGE = "a very distinguishable warning message";
+
+    private static final String SYS_OUT = "SystemOut";
+
     @Test
-    void julTest() throws InterruptedException {
+    void julInfoTest() throws IOException {
+        julLogger.log(Level.INFO, JUL_INFO_MESSAGE);
+        try {
+            Assert.assertTrue(checkMessage(".*INFO.*" + JUL_INFO_MESSAGE + ".*scopeInfo:.*"));
+        } catch (IOException e) {
+        }
+    }
+
+    @Test
+    void julWarnTest() throws IOException {
+        julLogger.log(Level.WARNING, JUL_WARN_MESSAGE);
+        try {
+            Assert.assertTrue(checkMessage(".*INFO.*" + JUL_WARN_MESSAGE + ".*scopeInfo:.*"));
```

Comment:
does OTel show "INFO" when we log a warning? would have expected WARN or WARNING